### PR TITLE
changed code style for libmscore

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

**Comment by igorkorsukov** [comment]
To the wrong repository, prematurely

**Comment by Jojo-Schmitz** [comment]
I though this was supposed to be done in a way that does not destroy the way how `git blame` enables us to find who change the code when and why?

**Comment by igorkorsukov** [comment]
> I though this was supposed to be done in a way that does not destroy the way how `git blame` enables us to find who change the code when and why?

The git may ignore some revisions in blame https://medium.com/moxio/ignoring-bulk-change-commits-with-git-blame-5d50835e45c2 

There will be a problem with the web interface of the github, we wrote a request to the github, they replied that they had put it in the queue for implementation.

**Comment by Jojo-Schmitz** [comment]
Same/similar problem with QtCreator I believe

**Comment by Howard-C** [comment]
What about moving methods to source files? I mean, I'm not a fan, but I'm more not a fan to not doing this while making methods multi-line in header files. Honestly, it's hideous.

**Comment by IsaacWeiss** [comment]
I would definitely rather not make the one-line setters and getters take up so much space.

File: libmscore/accidental.h [hunk 1]
```diff
@@ -26,114 +26,168 @@
...
-      static bool isMicrotonal(AccidentalType t)  { return t > AccidentalType::FLAT2; }
-
-      QString accessibleInfo() const override;
-      };
+    SymId sym;
+    qreal x;
+    SymElement(SymId _sym, qreal _x) : sym(_sym),
+        x(_x)
+    {
+    }
```
Comment (Jojo-Schmitz) [inline libmscore/accidental.h:62]: This here is a horrible waste of space

**Pull request closed without merging**